tools/Raven.Migrator/CollectionsInfo.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Collections.Generic;
+
+namespace Raven.Migrator
+{
+    public class CollectionsInfo
```

Comment:
Is this class expected to have more members in the future? (seems too much to have a class for holding a single primitive member)

Reply:
It is rarely used and is easier to convert to JSON this way